Content.Server/Interaction/InteractionSystem.cs

```diff
@@ -175,6 +175,10 @@ private void HandleDragDropRequestEvent(DragDropRequestEvent msg, EntitySessionE
 
             // trigger dragdrops on the dropped entity
             RaiseLocalEvent(msg.Dropped, interactionArgs);
+
+            if (interactionArgs.Handled)
+                return;
```

Comment:
I'm not sure on this, but I think the idea is that in principle, a given interaction event subscription / component interface can choose to ignore the handled status of the event and and do some interaction. So I think it's not supposed to return early. But I'm not sure about that, and I don't know if that functionality is ever actually used.

Reply:
Ah I only just realised that previously `DragDropEvent` wasn't even a `HandledEntityEventArgs` before this PR, so none of the interfaces would ever have even checked `args.Handled`.

Given that no new interfaces should ever be added, and that this is the first system that uses DragDropEvent & would be the only one that could potentially block interfaces from running their code its probably fine as is.

Reply:
just mimics and blocks behavior of the code with interfaces involved below it. (This is DragDrop specific, only to suppress Interface functions), what I can do: raise event both on the dropped and target before check, and ignore interfaces if it Handled?